### PR TITLE
Create newsletter_for_january.md

### PR DESCRIPTION
**Pull request by ziruier**
### Purpose

- No issue is linked to this PR.

### Changes

- No changes.

### Review Instructions

- No instructions.

### CODE REVIEW

**Comment by ziruier** [comment]
@neozhaoliang PTAL

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+烧风同学持续高产 ~ 这次的作品是一只水晶兔子，祝社区的小伙伴们兔年吉祥，健康平安！
+
+![ulRgAvKp_video(4)_1 compressed (2)](https://user-images.githubusercontent.com/124654014/217158864-6f596566-c875-4b73-9716-1902545567a2.gif)
+
+4k 分辨率，120 帧，512 spp，光追深度 512
+
+## 📢 社区活动
+
+[编译器优化技术 Equality Saturation，是如何提升运算性能的？](https://www.bilibili.com/video/BV1hG4y1w79u/?spm_id_from=333.999.0.0&vd_source=7e8cfbc83bcd0c8522627c6544d35724)
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:52]:
```suggestion
[编译器优化技术 Equality Saturation 是如何提升运算性能的？](https://www.bilibili.com/video/BV1hG4y1w79u/?spm_id_from=333.999.0.0&vd_source=7e8cfbc83bcd0c8522627c6544d35724)
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+本月合并 231 个 PR，解决 39 个 Issue。
+
+## 🌟 社区精选作品
+
+- Ti example 投稿：[Cornell box](https://github.com/HK-SHAO/RayTracingPBR/blob/taichi-dev/examples/cornell_box/cornell_box_shortest.py)。
+
+康奈尔盒子是图形学中常用来观察和测试全局光照的场景，社区的烧风同学使用蒙特卡洛路径追踪采样简化程序后只用了 139 行就完成了这个作品.
+  
+![编辑图片](https://user-images.githubusercontent.com/124654014/217157256-42f178f9-da22-41a9-ba31-d687f963adf4.jpeg)
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:40]:
```suggestion
![](https://user-images.githubusercontent.com/124654014/217157256-42f178f9-da22-41a9-ba31-d687f963adf4.jpeg)
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  📌 一月高光时刻\n Taichi 发布 v1.4 版本\Taichi Slack 将逐步迁移至 Discord\RAL 论文：基于弹性体形变仿真的视触觉传感器仿真器 Tacchi
+---
+
+# Taichi 2023 年 1 月社区月报
+
+## 📌一月高光时刻
+
+- Taichi v1.4 发布，Taichi AOT 正式发布 Vulkan 后端的运行时库 TiRT 及其 C API.
+
+- Taichi Slack 将逐步迁移至 Discord，[欢迎来讨论！](https://github.com/taichi-dev/taichi/issues/7206)
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:14]:
```suggestion
- Taichi Slack 将逐步迁移至 Discord，欢迎来[讨论](https://github.com/taichi-dev/taichi/issues/7206)！
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+论文一作陈子熙同学也将录制 Paper Reading 视频来详细解读，敬请关注！
+
+## 🧑‍💻 如何成为 Taichi 贡献者
+
+欢迎大家参与 Taichi 开发，这里有一些 issue 欢迎大家尝试解决：
+- GGUI - mouse wheel event #6938
+- Supporting tuple types as struct members #6856
+- Request for the feature to get the size of ndarray in Taichi kernel. #6799
+
+更多 good first issue（非常容易上手） 和  welcome contribution （稍微带有挑战性）的 issue [请查阅](https://github.com/taichi-dev/taichi/contribute)
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:73]:
```suggestion
更多 good first issue（非常容易上手）和  welcome contribution（稍微带有挑战性）的 issue 请查阅[这里](https://github.com/taichi-dev/taichi/contribute)
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+- **Taichi v1.4 发布！看看有哪些重要更新：**
+
+- Taichi AOT 已正式启用，并提供了基于 C API 的 Taichi 原生运行时库（TiRT）。现在不需要 Python 解释器也可运行编译好的 Taichi 内核。
+- Taichi kernel 编译速度显著提升，在一些大量使用 ti.Matrix 的场景中提升可达 2 倍.
+- Taichi ndarray 现已正式发布。这是一种保存连续多维数据的数组对象，可以方便地与外部库进行数据交换。
+- 支持所有后端的动态索引。不再需要 dynamic_index=True 来访问具有运行时变量的向量/矩阵。
+- 移除了 Metal 后端对 sparse SNode 的支持。
+
+有关此版本的更多更新，请参阅完整的 [变更日志](https://github.com/taichi-dev/taichi/releases)
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:28]:
```suggestion
有关此版本的更多更新，请参阅完整的[变更日志](https://github.com/taichi-dev/taichi/releases)
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+- Taichi Slack 将逐步迁移至 Discord，欢迎来[讨论](https://github.com/taichi-dev/taichi/issues/7206)！
+
+- RAL 论文：基于弹性体形变仿真的视触觉传感器仿真器 Tacchi.
+
+##  ⚙️ 技术动态
+
+- **Taichi v1.4 发布！看看有哪些重要更新：**
+
+- Taichi AOT 已正式启用，并提供了基于 C API 的 Taichi 原生运行时库（TiRT）。现在不需要 Python 解释器也可运行编译好的 Taichi 内核。
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:22]:
```suggestion
- Taichi AOT 已正式启用，并提供了基于 C API 的 Taichi 原生运行时库 (TiRT)。现在不需要 Python 解释器也可运行编译好的 Taichi 内核。
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+论文一作陈子熙同学也将录制 Paper Reading 视频来详细解读，敬请关注！
+
+## 🧑‍💻 如何成为 Taichi 贡献者
+
+欢迎大家参与 Taichi 开发，这里有一些 issue 欢迎大家尝试解决：
+- GGUI - mouse wheel event #6938
+- Supporting tuple types as struct members #6856
+- Request for the feature to get the size of ndarray in Taichi kernel. #6799
+
+更多 good first issue（非常容易上手）和  welcome contribution（稍微带有挑战性）的 issue 请查阅[这里](https://github.com/taichi-dev/taichi/contribute)。
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:73]:
```suggestion
更多 good first issue（非常容易上手）和 welcome contribution（稍微带有挑战性）的 issue 请查阅[这里](https://github.com/taichi-dev/taichi/contribute)。
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+赶快升级体验新功能吧 👉 `pip install -U taichi==1.4.1`
+
+本月合并 231 个 PR，解决 39 个 Issue。
+
+## 🌟 社区精选作品
+
+- Ti example 投稿：[Cornell box](https://github.com/HK-SHAO/RayTracingPBR/blob/taichi-dev/examples/cornell_box/cornell_box_shortest.py)。
+
+康奈尔盒子是图形学中常用来观察和测试全局光照的场景，社区的烧风同学使用蒙特卡洛路径追踪采样简化程序后只用了 139 行就完成了这个作品.
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:38]:
```suggestion
康奈尔盒子是图形学中常用来观察和测试全局光照的场景，社区的烧风同学使用蒙特卡洛路径追踪采样简化程序后只用了 139 行就完成了这个作品。
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+- Taichi Slack 将逐步迁移至 Discord，欢迎来[讨论](https://github.com/taichi-dev/taichi/issues/7206)！
+
+- RAL 论文：基于弹性体形变仿真的视触觉传感器仿真器 Tacchi.
+
+##  ⚙️ 技术动态
+
+- **Taichi v1.4 发布！看看有哪些重要更新：**
+
+- Taichi AOT 已正式启用，并提供了基于 C API 的 Taichi 原生运行时库 (TiRT)。现在不需要 Python 解释器也可运行编译好的 Taichi 内核。
+- Taichi kernel 编译速度显著提升，在一些大量使用 ti.Matrix 的场景中提升可达 2 倍.
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:23]:
```suggestion
- Taichi kernel 编译速度显著提升，在一些大量使用 ti.Matrix 的场景中提升可达 2 倍。
```

File: website/newsletter/zh-Hans/2023-01/newsletter_for_january.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+- Taichi ndarray 现已正式发布。这是一种保存连续多维数据的数组对象，可以方便地与外部库进行数据交换。
+- 支持所有后端的动态索引。不再需要 dynamic_index=True 来访问具有运行时变量的向量/矩阵。
+- 移除了 Metal 后端对 sparse SNode 的支持。
+
+有关此版本的更多更新，请参阅完整的[变更日志](https://github.com/taichi-dev/taichi/releases)
+
+赶快升级体验新功能吧 👉 `pip install -U taichi==1.4.1`
```
Comment (neozhaoliang) [inline website/newsletter/zh-Hans/2023-01/newsletter_for_january.md:30]:
```suggestion
赶快升级体验新功能吧 👉 `pip install -U taichi==1.4.1`.
```

**Review by neozhaoliang — APPROVED** [review]
LGTM!

**Pull request merged**